config/spotbugs-exclude.xml

```diff
@@ -45,6 +45,46 @@
       <Method name="main" />
       <Bug pattern="DM_EXIT" />
     </Match>
+    <Match>
+        <Class name="com.puppycrawl.tools.checkstyle.Main" />
+        <Method name="loadProperties" />
+        <Bug pattern="RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE" />
```

Comment:
please share in issue exact list of violations.

Reply:
Done.

@krichter722 , please keep addressing each review point, I replied points for us mean that you are still in progress.